src/librustc_builtin_macros/deriving/generic/mod.rs

```diff
@@ -818,7 +818,7 @@ impl<'a> MethodDef<'a> {
     ) -> P<Expr> {
         let substructure = Substructure {
             type_ident,
-            method_ident: cx.ident_of(self.name, trait_.span),
```

Comment:
is the `ident_of` method still used anywhere?

Reply:
Yes, lots of places. I might remove it in a future PR.